packages/grid/_modules_/grid/components/toolbar/GridToolbarFilterButton.tsx

```diff
@@ -76,9 +76,9 @@ export const GridToolbarFilterButton = React.forwardRef<
       <div>
         {apiRef!.current.getLocaleText('toolbarFiltersTooltipActive')(counter)}
         <ul className={classes.list}>
-          {activeFilters.map((item) => ({
+          {activeFilters.map((item, index) => ({
             ...(lookup[item.columnField!] && (
-              <li key={item.id}>
+              <li key={index}>
```

Comment:
+1 for reverting this change that Damien did recently.

Reply:
So keep that change right?

Reply:
Yeap